src/components/Button/Button.spec.js

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from "@testing-library/react";
+import Button from "./Button";
+
+describe("Button", () => {
+  it("renders a children text", () => {
+    render(<Button />);
+    const button = screen.getByRole("button", { children: "Home" });
+    expect(button).toBeInTheDocument();
+  });
+});
```

Comment:
I don't think this test works, right?
```suggestion
describe("Button", () => {
  it("renders a children text", () => {
    render(<Button>Click me</Button>);
    const button = screen.getByRole("button", { name: "Click me" });
    expect(button).toBeInTheDocument();
  });
});
```